spec/features/viewing_application_templates_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'rails_helper'
+
+describe 'viewing application forms on dashboard' do
+  let!(:position) { create :position }
+  let!(:user) { create :user, staff: true }
+  let!(:app_with_draft) { create :application_template }
+  let!(:app_without_draft) { create :application_template }
+  let!(:draft) do
+    create :application_draft,
+           application_template: app_with_draft,
+           user: user
+  end
+  before :each do
+    when_current_user_is user, integration: true
+    visit staff_dashboard_url
+  end
+  it 'has links to view the applications' do
+    click_link 'View application',
+               href: application_path(app_with_draft)
+    expect(page.current_url)
+      .to include application_path(app_with_draft)
```

Comment:
Oops. Because I had to do it one time due to the full URL including some parameters, and then got into the habit of it apparently.